engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/deletion/content-view-version-deletion-confirm.controller.js

```diff
@@ -51,24 +69,8 @@ angular.module('Bastion.content-views').controller('ContentViewVersionDeletionCo
                 params['system_environment_id'] = $scope.deleteOptions.contentHosts.environment.id;
             }
 
+            /*eslint new-cap:0*/
             ContentView.removeAssociations(params, success, error);
```

Comment:
http://eslint.org/docs/rules/new-cap.html  -- it is currently configured to treat `CamelCase` as meaning an instantiable class type of object rather than a 'Module' like object with class functions that we might have in other languages.